drivers/mbox/Kconfig.nxp_mailbox

```diff
@@ -4,6 +4,6 @@
 config MBOX_NXP_MAILBOX
```

Comment:
The usual commit prefix looks like this: ``drivers: mbox: ``.

other than that looks good to me.

Reply:
Thanks, updated :)